src/devices/Directory.Build.props

```diff
@@ -10,4 +10,8 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
   </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="UnitsNet" Version="4.58.0" />
```

Comment:
NIT: should we instead just add this to the projects that actually require it? I think that is cleaner. Also the version we depend on should be moved to Versions.props.

Reply:
Iot.Device.Bindings depends on it anyway, for all tiny project this won't change anything anywaym this way we don't need to pollute csprojes... Once (if) we split projects into packages then I'd change that

Reply:
I'm with krwq here. Once this is properly used, there will be few projects _not_ requiring it (maybe some output devices).

Reply:
That's fair, I'm fine with keeping it here then. That said, I do think we still want to move the version to Versions.props on this PR since it is actually used in a few places so we want to centralize the version.